app/assets/javascripts/components/expander.js

```diff
@@ -42,8 +39,16 @@ window.GOVUK.Modules = window.GOVUK.Modules || {}; // if this ; is omitted, none
   Expander.prototype.bindChangeEvents = function (e) {
     for (var i = 0; i < this.$allInteractiveElements.length; i++) {
       var $el = this.$allInteractiveElements[i]
+      if ($el.tagName === 'SELECT') {
+        $el.addEventListener('change', this.updateSelectedCount.bind(this))
+      }
+
       // but for inputs we need both change and enter key event
       if ($el.tagName === 'INPUT') {
+        if ($el.type === 'text') {
+          $el.addEventListener('change', this.updateSelectedCount.bind(this))
+        }
```

Comment:
I think this `if` statement isn't necessary - if you follow the logic of the code if this if statement is true and executes, lines 52 and 53 run regardless - and line 49 appears to be identical to line 52, so it runs twice.

Reply:
good shout, line 49 and 52 aren't quite identical but the `handleInputEvent` on 52 calls `updateSelectedCount` so we're all good without the `if` :) 🎉